dropbox_to_local.py

```diff
@@ -0,0 +1,22 @@
+import dropbox
+import os
+
+dbx = dropbox.Dropbox('OAUTH_API_CREDENTIALS_HERE')
+dbx.users_get_current_account()
+
+lp = "/local/directory/path" # local directory path
+r = "/folder_name" # directory in dropbox to download
+
+def download_files(local_path, root):
+	for entry in dbx.files_list_folder(root).entries:
+		if entry.__class__ == dropbox.files.FolderMetadata:
```

Comment:
more idiomatic to use `cls(entry)` instead

Reply:
not sure what you meant by this actually, dm'ed you the question via slack!